tests/ZendTest/Feed/Reader/StandaloneExtensionManagerTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+namespace ZendTest\Feed\Reader;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Zend\Feed\Reader\StandaloneExtensionManager;
+
+class StandaloneExtensionManagerTest extends TestCase
+{
+    public function setUp()
+    {
+        $this->extensions = new StandaloneExtensionManager();
```

Comment:
No, we don't. We often omit the property declarations and just use public properties in the test suite.